content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -114,6 +114,17 @@ to the name of the namespace. You can't change this label's value.
 This is useful if you want to target a specific namespace with a label
 {{< glossary_tooltip text="selector" term_id="selector" >}}.
 
+### kubernetes.io/bootstrapping
+
+Example: ` kubernetes.io/bootstrapping=rbac-defaults`
+
+Used on: Clusters
```

Comment:
I don't think `Used on: Clusters` is correct
```suggestion
Used on: Clusters
```

Reply:
@reylejano the following page (https://docs.openshift.com/container-platform/4.7/authentication/using-rbac.html) demonstrates that this annotation is used to view the cluster roles etc.

Reply:
ClusterRoles and Clusters are not the same

Reply:
Do you have any idea from where I can learn about this annotation?

Reply:
- https://github.com/kubernetes/kubernetes has the source code
- you could ask SIG Auth folks
- it's also OK to leave this for someone who is already very familiar with RBAC in Kubernetes

Reply:
a couple tweaks needed:

1. this is a label, not an annotation
2. this is added to default Role / ClusterRole / RoleBinding / ClusterRoleBinding objects created by the `kube-apiserver`, not `Cluster` objects